documentation/userguide/docs/metadata_forms.md

```diff
@@ -114,3 +113,17 @@ After user clicks this button and selects the available form from drop-dow list,
 After all required fields are filled, press "Attach" button in the right upper corner of the editing area.
 
 ![metadata form attach](pictures/metadata_forms/attach_mf.png#zoom#shadow)
+
+
+**Permissions Summary**
+1. **Create Metadata Form:** User given the permission from data.all administrators can create metadata forms with the Global visibility and visibility for their teams. Owners and Admins of Organizations and Environments can create metadata forms with Environment-Wide and Organization-wide visibility for owned entities.
+2. **Edit and delete Metadata Form:** This actions available only for owners of the metadata form.
+3. **Enforce Metadata Form Usage**: 
+* Data.all admins can enforce any form on any level across the platform. They have full control over metadata form enforcement.
+* Owners/admins  of a data.all entity can enforce forms for this levels and levels below in the hierarchy. For example, an org admin can enforce a form for the org, all teams in that org, all environments in the org, all datasets in those environments, etc.
+* Share approvers and requestors can enforce forms for a specific share they are involved with. However, they can only delete enforcement rules they created themselves - they cannot delete rules created by others
+4. **Attach Metadata Form**: 
+* Users must have relevant  permissions for target entity (given by entity admin)
```

Comment:
what are the relevant permissions? is it GET_DATASET or is it ATTACH_METADATA_FORM_TO_DATASET?